lib/fluent/log/console_adapter.rb

```diff
@@ -0,0 +1,66 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'console/serialized/logger'
```

Comment:
```suggestion
require 'console/terminal/logger'
```

`Console::Terminal::Logger` is defined in https://github.com/socketry/console/blob/main/lib/console/terminal/logger.rb.

It seems to work without this `require` somehow...

Reply:
Oops, I've fixed it. Thanks!